MANIFEST.in

```diff
@@ -1,6 +1,10 @@
+include .coveragerc
+include .editorconfig
```

Comment:
IMO it makes no sense to include .editorconfig in the distribution. You only need it if you want to hack the source, but therefore you need a clone of the repository and not the distribution.